egs/hub4_spanish/s5/run.sh

```diff
@@ -0,0 +1,316 @@
+    exp/tri2 data/local/dictp/tri2 data/local/langp/tri2 data/langp/tri2
+fi
+
+if [ $stage -le 8 ]; then
+  echo ---------------------------------------------------------------------
+  echo "Stage 8: Starting (full) triphone training in exp/tri3 on" `date`
+  echo ---------------------------------------------------------------------
+  steps/align_si.sh \
+    --boost-silence $boost_sil --nj $train_nj --cmd "$train_cmd" \
+    data/train data/langp/tri2 exp/tri2 exp/tri2_ali
+
+  steps/train_deltas.sh \
+    --boost-silence $boost_sil --cmd "$train_cmd" \
+    $numLeavesTri3 $numGaussTri3 data/train data/langp/tri2 exp/tri2_ali exp/tri3
+
+  local/reestimate_langp.sh --cmd "$train_cmd" --unk "$unk" \
+    data/train data/lang data/local/ \
+    exp/tri3 data/local/dictp/tri3 data/local/langp/tri3 data/langp/tri3
+fi
+
+if [ $stage -le 9 ]; then
+  echo ---------------------------------------------------------------------
+  echo "Stage 9: Starting (lda_mllt) triphone training in exp/tri4 on" `date`
+  echo ---------------------------------------------------------------------
+  steps/align_si.sh \
+    --boost-silence $boost_sil --nj $train_nj --cmd "$train_cmd" \
+    data/train data/langp/tri3 exp/tri3 exp/tri3_ali
+
+  steps/train_lda_mllt.sh \
+    --boost-silence $boost_sil --cmd "$train_cmd" \
+    $numLeavesMLLT $numGaussMLLT data/train data/langp/tri3 exp/tri3_ali exp/tri4
+
+  local/reestimate_langp.sh --cmd "$train_cmd" --unk "$unk" \
+    data/train data/lang data/local \
+    exp/tri4 data/local/dictp/tri4 data/local/langp/tri4 data/langp/tri4
+fi
+
+if [ $stage -le 10 ]; then
+  echo ---------------------------------------------------------------------
+  echo "Stage 10: Starting (SAT) triphone training in exp/tri5 on" `date`
+  echo ---------------------------------------------------------------------
+
+  steps/align_si.sh \
+    --boost-silence $boost_sil --nj $train_nj --cmd "$train_cmd" \
+    data/train data/langp/tri4 exp/tri4 exp/tri4_ali
+
+  steps/train_sat.sh \
+    --boost-silence $boost_sil --cmd "$train_cmd" \
+    $numLeavesSAT $numGaussSAT data/train data/langp/tri4 exp/tri4_ali exp/tri5
+
+  local/reestimate_langp.sh --cmd "$train_cmd" --unk "$unk" \
+    data/train data/lang data/local \
+    exp/tri5 data/local/dictp/tri5 data/local/langp/tri5 data/langp/tri5
+fi
+
+
+if [ $stage -le 11 ]; then
+  echo ---------------------------------------------------------------------
+  echo "Stage 11: Starting exp/tri5_ali on" `date`
+  echo ---------------------------------------------------------------------
+  steps/align_fmllr.sh \
+    --boost-silence $boost_sil --nj $train_nj --cmd "$train_cmd" \
+    data/train data/langp/tri5 exp/tri5 exp/tri5_ali
+
+  local/reestimate_langp.sh --cmd "$train_cmd" --unk "$unk" \
+    data/train data/lang data/local \
+    exp/tri5_ali data/local/dictp/tri5_ali data/local/langp/tri5_ali data/langp/tri5_ali
+fi
+
+
+if [ $stage -le 12 ]; then
```

Comment:
Yes, I agree -- I just wasn't sure if it will be the case given the fact we have ~30 hrs of data.
But I can remove it or move to a separate script.
y.